core/src/com/unciv/models/ruleset/Building.kt

```diff
@@ -355,7 +382,7 @@ class Building : NamedStats(), INonPerpetualConstruction, ICivilopediaText {
 
     override fun getStatBuyCost(cityInfo: CityInfo, stat: Stat): Int? {
         var cost = getBaseBuyCost(cityInfo, stat)?.toDouble()
-        if (cost == null) return null
+        if (cost == null) { println(name); return null}
```

Comment:
I have a little project here just cleaning up console output... I think we should try to:
- Have all debug output gated. Where and how? Uh, kotlin refuses to implement conditional compilation - argument if (const val Boolean) will optimize unused code away - that would mean the way I did it in Sounds is OK (except I left it on after it served its purpose).
- No println() - looks ugly in Logcat output
- No println(name) without some context hint...

Dumb idea?